css/style.css

```diff
@@ -71,11 +73,18 @@
 }
 
 .sidebar a {
-    border-bottom: 2px solid #f3e6b0;
-    padding-bottom: 0.5rem;
+  border-bottom: 2px solid #f3e6b0;
+  padding-bottom: 0.5rem;
+  font-size: 1.1rem;
+}
+
+.hd-3 {
+  margin: 1rem 0 0.2rem 0;
```

Comment:
if margin property has 3 values, you can add like this:- margin: 1rem 0 0.2rem;

Reply:
Ok! noted.
thank you for this